src/main/mw.c

```diff
@@ -236,14 +237,14 @@ void annexCode(void)
     }
 
     if (feature(FEATURE_VBAT)) {
-        if ((int32_t)(currentTime - vbatLastServiced) >= VBATINTERVAL) {
+        if (cmp32(currentTime, vbatLastServiced) >= VBATINTERVAL) {
```

Comment:
Maybe use better name for timestamp comparisons? 
`int32_t timestamp32Diff(int32_t, int32_t)` ?